code/__DEFINES/MC.dm

```diff
@@ -79,3 +79,14 @@
 	PreInit();\
 }\
 /datum/controller/subsystem/processing/##X
+
+///RunQueue() didnt complete a run and something fucked up
+#define RUNQUEUE_ERRORED FALSE
+///RunQueue() successfully completed a run
+#define RUNQUEUE_SUCCESSFUL_RUN TRUE
+
+///CheckQueue() wasnt able to go through its list of subsystems and something fucked up
+#define CHECKQUEUE_ERRORED FALSE
+///CheckQueue() was able to go through its list of subsystems
+#define CHECKQUEUE_SUCCESSFUL_RUN TRUE
```

Comment:
Why not just TRUE/FALSE?

Reply:
eh probably better to keep it to TRUE/FALSE i just thought they were important enough to warrant the define

Reply:
It's the same pattern as most other procs that succeed/fail right?